sdk-python/create_bulk_messages.py

```diff
@@ -0,0 +1,83 @@
+import argparse
+import logging
+import os
+import uuid
+
+from awaazde import AwaazDeAPI
+from awaazde.constants import CommonConstants
+from awaazde.utils import CSVUtils
+
+
+def parse_arguments():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("username", help="Username", type=str)
+    parser.add_argument("password", help="Password", type=str)
+    parser.add_argument("organization", help="Organization ", type=str)
+    parser.add_argument("csv_file_path", help="Path for the csv ", type=str)
+    parser.add_argument('--use_custom_format', type=bool, default=False,
+                        help="Set this as True if the messages are of a custom Xact Template")
+    args = parser.parse_args()
+    return args
+
+
+def get_next_tag(headers):
+    """
+    :param headers: list of headers
+    :type headers: list
+    :return: The next tag field, if the headers has tag1,tag2 returns tag3, If no tag field present returns "tags"
+    :rtype: String
+    """
+    tags = [x for x in headers if x.startswith(CommonConstants.TAG_FIELD)]
+    tags = sorted(tags, reverse=True)
+    if len(tags) > 0:
+        tag_number = int(filter(str.isdigit, str(tags[0])))
+        return 'tag{}'.format(tag_number + 1)
+    else:
+        return CommonConstants.TAGS_FIELD
+
+
+def create_bulk_messages(headers, message_data, file_path, **kwargs):
+    """
+    :param headers: Headers of the data
+    :type headers: List
+    :param message_data: Message data eg: [{phone_number:8929292929,send_on:"",tag1:"tag_number1",template:23,language:"hi"}]
+    :type message_data: List of dict
+    :param file_path: Path to the folder where the file with created/pending messages are to be kept
+    :type file_path: String
+    :return: Creates another csv_files in the folder where the csv file was present
+            with created and not created messages
+    """
+    message_request_id = str(uuid.uuid1())
```

Comment:
I believe this functionality is no longer required. And if it is required, user should add it in the file upload directly